README.md

```
@@ -48,3 +52,79 @@ This contract manages the escrow mechanism essential for secure trading:
 ## [Deployment/Development instructions here](INSTRUCTIONS.md)
 
 ---
+
+## Review process and merge criteria
+
+### Process overview
+
+1. Follow the instructions below to create a PR that would [Adding a price feed address to the list](#Adding-a-price-feed-address-to-the-list) or [Adding an authorization address to the list](#Adding-an-authorization-address-to-the-list).
```

Markdownlint

Expected: #adding-a-price-feed-address-to-the-list; Actual: #Adding-a-price-feed-address-to-the-list
Link fragments should be valid

(MD051, link-fragments)